client/src/app/util/__tests__/generateImageSpec.js

```diff
@@ -29,6 +30,14 @@ describe('util - generateImage', function() {
       expect(image).to.exist;
     });
 
+
+    it('should automatically handle large images <' + type + '>', function() {
+
+      const image = generateImage(type, outOfBoundsSVG);
+
+      expect(image.length).to.be.greaterThan(6);
```

Comment:
Just for understanding: what means 'greater than 6' here?

Reply:
Ah yeah, maybe I should put a comment here. Normaly if an image cannot be generated it returns a 6 character long data, something like `data=;,`, if it can be generated it returns the actualy size of the image (depending on how many bytes is it, some big number)

Reply:
+1 for comment